### ISSUE TITLE
for climber import, provide a "dry run" capability

### ISSUE THREAD

**Issue by badpens** [jsnyders/ClimbingComp#5]
would be nice to have a test-but-don't-do capability for the import function.

that way, you could test an import without actually changing the db, and see if the stats make sense.

for example, if you import the lastest master climber db a couple weeks after the first comp, you can see that for the most part not much changed, just some small number of climbers were added.

after the user checks the stats, they could click the "do it for real this time" button.

**Comment by jsnyders** [comment]
Good idea. I have started to implement this. For now the "do it for real" button is simply doing the import again and choose something other than test. I'll leve this open until the feature gets more testing.
